### ISSUE TITLE
License handling does not work

### ISSUE THREAD

**Issue by mpoberezhniy** [megastep/makeself#326]
The error occurs when trying to use `--license`.
```
./licensed_makeself_bundle.run
more: unknown option -e
Try 'more --help' for more information.
Please type y to accept, n otherwise: n
```

Environment:
```
RHEL 8.6:
Linux hostname 4.18.0-372.32.1.el8_6.x86_64 #1 SMP Fri Oct 7 12:35:10 EDT 2022 x86_64 x86_64 x86_64 GNU/Linux
```

Ubuntu 20.04:
```
uname -r
5.15.0-101-generic

more -e
more: bad usage
Try 'more --help' for more information.
```

**Comment by mpoberezhniy** [comment]
Looks like the more -e option is not wide-spread yet, probably some error handling should be implemented.

**Comment by mpoberezhniy** [comment]
Fixed with #329
